lib/extend.js

```diff
@@ -12,13 +12,38 @@ const extend = {
         return {exposes, fromZigbee, toZigbee};
     },
     light_onoff_brightness: (options={}) => {
-        options = {disableEffect: false, ...options};
```

Comment:
Could you revert this formatting changes?

Reply:
@Koenkk for options only or for the whole file?

Reply:
@Koenkk there are 6 items + 2 ternary operators inside the `toZigbee` array - it is starting to get a bit hard to read 😅